app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -859,6 +869,35 @@ private void passInfoToSheet(Place place) {
         commonsButton.setVisibility(this.place.hasCommonsLink()?View.VISIBLE:View.GONE);
         commonsButton.setOnClickListener(view -> openWebView(this.place.siteLinks.getCommonsLink()));
 
+        discussionButton.setOnClickListener(v -> {
+            Intent intent=new Intent(this.getActivity(), WikiFeedback.class);
+            intent.putExtra("place",this.place.name);
+            Timber.d("line866"+ this.place.name);
+            Observable.fromCallable(() -> nearbyController.getFeedback(this.place.name))
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(result -> {
+                        if (result!=null){
+                            Timber.d("line871"+result);
+                            intent.putExtra("wikidataEntry",result);
+                            startActivity(intent);
+                        }
+                        else {Toast.makeText(this.getActivity(),"Failed",Toast.LENGTH_SHORT).show();
+                                                    }
+                    }, throwable -> {
+
+                        Timber.e(throwable, "Error occurred while loading notifications");
+                        throwable.printStackTrace();
+                    });
+            /*try {
+                intent.putExtra("wikidataEntry",this.place.name);
+                startActivity(intent);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }*/
```

Comment:
Is this intended?

Reply:
No that was added mistakenly I have removed it